app/scripts/contentscript.js

```diff
@@ -46,15 +43,6 @@ function injectScript(content) {
   }
 }
 
-/**
- * Sets up the stream communication and submits site metadata
- *
- */
-async function start() {
-  await setupStreams()
-  await domIsReady()
```

Comment:
I think we needed it in the past because we pulled metadata in this script instead of in the inpage provider. Now, we were just calling `start`, which first set up the streams, and then waited for the DOM to be ready to do, nothing.